### PR TITLE
feat: implement experimental dnt setting

### LINKED ISSUE #384: Add option to enable "Do not Track" header
**Is your feature request related to a problem? Please describe.**
The do not track option in most modern browsers shows websites a user does not want to be tracked, resulting in slightly better privacy. I'd like to use this feature, but sadly it is not included in wexond

**Describe the solution you'd like**
Implement a checkbox in the settings affecting both the DNT-header as well as the JavaScript function "navigator.doNotTrack" (both testable at [https://browserleaks.com/donottrack](https://browserleaks.com/donottrack))

**Describe alternatives you've considered**
None.

**Additional context**
None.

### PR DESCRIPTION
**Pull request by Fyers**
This PR implements DNT-Settings support (closes #384)

Setting the header in `sessions-manager` with `session.defaultSession` did not work as expected.

### CODE REVIEW

File: src/main/view.ts [hunk 1]
```diff
@@ -51,6 +52,14 @@ export class View extends BrowserView {
     this.window = window;
     this.homeUrl = url;
 
+    const { object: settings } = windowsManager.settings;
+    if (settings.doNotTrack) {
```
Comment (sentialx) [inline src/main/view.ts]: The `if` could be moved into the handler, so that it always updates when the setting changes.

**Review by sentialx — CHANGES_REQUESTED** [review]

File: src/main/view.ts (hunk 1 above)
Comment (Fyers) [reply src/main/view.ts]: > The `if` could be moved into the handler, so that it always updates when the setting changes.

which handler do you mean?

Comment (sentialx) [reply src/main/view.ts]: Inside the `onBeforeSendHeaders` handler.

**Comment by sentialx** [comment]
I'll soon merge your PRs, once I will get a CLA.

**Comment by sentialx** [comment]
recheckcla

**Comment by sentialx** [comment]
recheckcla

**Comment by sentialx** [comment]
**CLA Assistant Lite:** <br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/wexond/desktop/blob/master/cla.md) before we can accept your contribution. You can sign the CLA by just  posting a Pull Request Comment same as the below format.
  - - -
  ***I have read the CLA Document and I hereby sign the CLA***
  - - - 
  <sub>You can retrigger the CLA Action by commenting **recheckcla** in this Pull Request</sub>

**Comment by sentialx** [comment]
recheckcla

**Comment by Fyers** [comment]
I have read the CLA Document and I hereby sign the CLA

**Pull request merged**